ee/tabby-ui/app/(dashboard)/components/sidebar.tsx

```diff
@@ -96,6 +96,9 @@ export default function Sidebar({ children, className }: SidebarProps) {
                     <SidebarButton href="/settings/git">
                       Git Providers
                     </SidebarButton>
+                    <SidebarButton href="/settings/gitops">
+                      GitOps
+                    </SidebarButton>
```

Comment:
```suggestion
```

Merge this into `Git Providers` as an new tab (e.g `Git` / `GitHub`). Could do it in a follow up pr